docs/docs/v7-upgrade-guidance.mdx

```diff
@@ -431,12 +433,15 @@ Command|Old option|New option
 [aad group get](./cmd/aad/group/group-get.mdx)|`title`|`displayName`
 [aad user set](./cmd/aad/user/user-set.mdx)|`objectId`|`id`
 [aad user set](./cmd/aad/user/user-set.mdx)|`userPrincipalName`|`userName`
+[flow connector export](./cmd/pa/connector/connector-export.mdx)|`connector`|`name`
```

Comment:
isn't this already mentioned below as this is the `pa connector export` 🤔?

Reply:
Well, but still we need to mention it. Someone might miss it and not update his scripts because he thinks he doesn't use the command.

Reply:
hmm. I don't think I follow 😔. But in line 444 we already mention 
```
[pa connector export](./cmd/pa/connector/connector-export.mdx)|`connector`|`name`
```

isn't it good enough?

Reply:
Hi @Adam-it, I'll explain some more: so say someone uses the flow export command on his script, and he does not know it's an alias of pa export, he might be confused and not update his scripts...

Reply:
Which is why we would need to reference both 'commands'

Reply:
aaa. clear. 👍Sorry for the stupid question. Let's leave it as is